Build/Props/CodeJam.Targeting.props

```diff
@@ -11,8 +11,8 @@
 	<PropertyGroup>
 		<CopyMeMinimalTargetFrameworks>net461;netstandard2.0</CopyMeMinimalTargetFrameworks>
 		<CopyMeMinimalTestTargetFrameworks>net461;netcoreapp2.0</CopyMeMinimalTestTargetFrameworks>
-		<CopyMeTargetFrameworks>net472;net461;net45;net40;net35;net20;netstandard2.1;netstandard2.0;netstandard1.5;netstandard1.3;netcoreapp3.0;netcoreapp2.0;netcoreapp1.0</CopyMeTargetFrameworks>
-		<CopyMeTestTargetFrameworks>net48;net472;net471;net47;net461;net45;net40;net35;net20;netcoreapp3.0;netcoreapp2.1;netcoreapp1.1</CopyMeTestTargetFrameworks>
+		<CopyMeTargetFrameworks>net50;net472;net461;net45;net40;net35;net20;netstandard2.1;netstandard2.0;netstandard1.5;netstandard1.3;netcoreapp3.0;netcoreapp2.0;netcoreapp1.0</CopyMeTargetFrameworks>
```

Comment:
We need to regenerate FW monikers in CodeJam.TargetingTests.EmitTargetingConstants

Reply:
Sure, will add there.

Reply:
Actually didn't know it is autogenerated :)
Why not T4 as done in other places ?

Reply:
Let's fix it after this PR.